templates/zerver/emails/digest.html

```diff
@@ -17,25 +17,25 @@
                         {% endfor %}
                     </div>
                 </div>
-                {% if convo.count > 0 %}<p>+ {{ convo.count }} more message{{ convo.count|pluralize }} by {{ convo.participants|display_list(4) }}.</p>{% endif %}
+                {% if convo.count > 0 %}<p class="digest_paragraph">+ {{ convo.count }} more message{{ convo.count|pluralize }} by {{ convo.participants|display_list(4) }}.</p>{% endif %}
```

Comment:
It seemed safer for me to add a class to the <p> tags in this file, rather than writing hard CSS selectors.

Reply:
Yeah that seems reasonable.